samples/common/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    id("com.android.library")
+    id("kotlin-android")
+    id("kotlin-parcelize")
+    id("appyx-publish-android")
```

Comment:
We don't want to publish this I assume? you should remove this, otherwise we will accidentally publish it next release.

Reply:
Good catch!! Copy-paste remnant.